### PR TITLE
Feature/ first tests for routes

### PR DESCRIPTION
**Pull request by NikPaukov**

### CODE REVIEW

File: src/repositories/likes.repository.ts [hunk 1]
```diff
@@ -35,5 +35,5 @@ export const updateOne = (
   id: string,
   newData: { tweet?: string; user?: string }
 ) => {
-  LikeModel.updateOne({ _id: id }, newData);
+  return LikeModel.findOneAndUpdate({ _id: id }, newData);
```
Comment (VitaliiDenysiuk) [inline src/repositories/likes.repository.ts]:
```suggestion
  return LikeModel.findOneAndUpdate({ _id: id }, newData, {
  new: true
});
```
You should set the new option to true to return the document after update was applied.
https://mongoosejs.com/docs/tutorials/findoneandupdate.html

File: src/test/routes.test.ts [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+            password: "1234567",
+            email: "zzz@aaaaa.aaa",
+            name: "aaa",
+          }).expect(400);
+        });
+
+        it("shouldn`t work with blank username/email/name", async () => {
+          await register({
+            username: "",
+            password: "1234567",
```
Comment (VitaliiDenysiuk) [inline src/test/routes.test.ts]: You can have mistake with length of password here. if somebody change order of check

File: src/test/routes.test.ts [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+            email: "zzz@aaaaa.aaa",
+            name: "aaa",
+          });
+          await register({
+            username: "zzz",
+            password: "1234567",
```
Comment (VitaliiDenysiuk) [inline src/test/routes.test.ts]: same

File: src/test/routes.test.ts [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+            email: "",
+            name: "aaa",
+          }).expect(400);
+          await register({
+            username: "zz",
+            password: "1234567",
```
Comment (VitaliiDenysiuk) [inline src/test/routes.test.ts]: same

File: src/test/routes.test.ts [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+            .send({ refreshToken: refreshToken })
+            .expect(200);
+          expect(resp.body.accessToken).not.toBeUndefined();
+        });
+        it("shouldn`t work with invalid token", async () => {
+          const resp = await request(app)
+            .post("/api/auth/refresh")
+            .send({ refreshToken: refreshToken + "a" })
+            .expect(400);
+          expect(resp.body.error).toEqual("invalid signature");
```
Comment (VitaliiDenysiuk) [inline src/test/routes.test.ts:147]: I check response:
{
    "error": "invalid token"
}

File: src/test/routes.test.ts [hunk 6]
```diff
@@ -0,0 +1,360 @@
...
+let id: string;
+const register = (data: object) => {
+  return request(app).post("/api/auth/register").send(data);
+};
+describe("Routes tests", () => {
+  describe("User routes", () => {
+    describe("Auth routes", () => {
+      describe("Register", () => {
+        it("should work", async () => {
+          await register({
```
Comment (VitaliiDenysiuk) [inline src/test/routes.test.ts]: we can get problem if previous test was bad at position "delete" and such user was at database

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
very good job done.
fix some remarks

File: src/test/routes.test.ts (hunk 5 above)
Comment (NikPaukov) [reply src/test/routes.test.ts:147]: image

Comment (VitaliiDenysiuk) [reply src/test/routes.test.ts:147]: it's meaning we can have two different error. If I remember well I added at the beginning of the token letter

**Pull request merged**